proto/README.md

```diff
@@ -0,0 +1,23 @@
+## How to compile protos + use locally
+
+1. Get the latest proto [release](https://github.com/protocolbuffers/protobuf/releases) (select one with python included if you want to run this validator)
+1. Install the [C++ Protocol Buffer Runtime](https://github.com/protocolbuffers/protobuf/blob/master/src/README.md)
```

Comment:
there don't happy to be any `brew install <magic package name>` shortcuts are there?

i.e. is what you're linking to the same as https://formulae.brew.sh/formula/protobuf?

Reply:
Good call, I'll add `brew install protobuf` to the instructions.